### DISCUSSION TITLE
Group hosts in Servers view

### DISCUSSION THREAD

**Discussion by MarkLFT** [gravitational/teleport#15346] [Q&A] [answered]
Is there anyway to refine the look of the list of hosts?  

I have many servers with the same host names, that are on different sites.  I have set labels to indicate which site each host belongs to, but it is annoying having to search up and down the list of tags to find the server I am after.  Is it possible to perhaps have some sort of hierarchical tree of sites and hosts?

**Comment by webvictim** [comment, marked as answer]
There's no hierarchical tree at the moment but if you click a label, it'll show only servers which have that label. You can sort the columns by hostname etc to get a better view. You can also type the name of the site in the search bar and press enter to filter the list.
